openmetadata-service/src/main/java/org/openmetadata/service/resources/system/ConfigResource.java

```diff
@@ -139,7 +139,7 @@ public SlackChatConfiguration getSlackChatConfiguration() {
   }
 
   @GET
-  @Path(("/airflow"))
+  @Path(("/pipelineServiceClient"))
```

Comment:
set a more generic name. Also updated on the UI side

Reply:
nit: IMO I would use a hyphen for the spaces like `pipeline-service-client`.